src/main/kotlin/no/nav/familie/ba/sak/integrasjoner/økonomi/UtbetalingsTidslinje.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ba.sak.integrasjoner.økonomi
+
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsperiode
+import no.nav.familie.tidslinje.Tidslinje
+import no.nav.familie.tidslinje.utvidelser.tilPerioderIkkeNull
+import java.math.BigDecimal
+import java.time.LocalDate
+
+class UtbetalingsTidslinje(
```

Comment:
Ingen stor sak, men navnet flyter kanskje litt bedre om man omdøper det til:

```suggestion
class UtbetalingTidslinje(
```

```suggestion
class Utbetalingstidslinje(
```

Ser det hetere `UtbetalingsperiodeDto` under